genie-docs/src/docs/asciidoc/_metrics.adoc

```diff
@@ -6,13 +6,7 @@
 |Time taken initialize the job environment (working directory, script) and fork the children
 |nanoseconds
 |JobLauncher
-|-
-
-|genie.jobs.submit.exception
-|Counts the number of jobs that failed to initialize and launch
-|count
-|JobLauncher
-|exceptionClass
+|status, exceptionClass
```

Comment:
Do we want the tag to be `exceptionClass` or would `exception` be sufficient. Just to save storage space in Atlas those few characters over and over can save a bunch of space.